xep-0156.xml

```diff
@@ -212,8 +179,9 @@ _xmppconnect IN TXT "_xmpp-client-websocket=wss://web.example.com:443/ws"
   <section2 topic='Business Rules' anchor='httpbizrules'>
     <p>The following business rules apply:</p>
     <ol start='1'>
+      <li>host-meta files MUST be fetched only over HTTPS, and MUST ignore any connection URLs that are not secure, eg not 'https://' or 'wss://', this provides secure delegation, meaning you SHOULD send SNI matching the host of the URL from the connection URL and validate that the certificate is valid for that host *or* the the XMPP domain</li>  
```

Comment:
Instead of having (negated) positive examples, I think it would be better to make explicit negative examples: "e.g. 'http://' or 'ws://'", otherwise uneducated users may decide they consider http without TLS secure enough.

nits:
- "and **clients** MUST ignore any..."
- "e.g." instead of "eg"
- Add full stop before "this provides secure delegation"
- "the the"
- Add full stop at end of item

Reply:
Thanks for the feedback, I've re-worded it and fixed the other issues you identified, but I really want to keep the "allow these" wording vs "do not allow these" as it encourages implementations to implement a list of allowed protocols which is more secure than a list of disallowed ones.

Reply:
Yes, this way is even better, given that as of now, https and wss are effectively the only schemes that should be supported, as bosh and websockets are the only alternative connection methods you can discover with the XEP.

Only other nit:
The markdownish `*or*` doesn't really fit here. We have `<em>or</em>` or `<strong>or</strong>` for this. :)